### PR TITLE
Install media corrections

### PR DESCRIPTION
**Pull request by lzap**
Couple of issues solved:

1) The chapter uses "third-party" extensively, this does not make any sense upstream and I don't believe it's important to have even downstream.

2) Added note about CentOS mirror changes which are upcoming: https://community.theforeman.org/t/rfc-rename-centos-mirror-installation-media/18867

3) Moved the chapter closer to syncing, IM + syncing are essentially very important and for upstream users without Katello this should really come earlier.

The move is a separate commit so you can easily review the changes.

### CODE REVIEW

File: guides/doc-Provisioning_Guide/topics/Resources.adoc [hunk 1]
```diff
@@ -198,6 +198,54 @@ To view the kickstart tree enter the following command:
 # hammer medium list --organization "_your_organization_"
 ----
 
+[[Configuring_Provisioning_Resources-Creating_Installation_Media]]
+=== Using Installation Media
+
+Installation media are sources of packages that {ProjectServer} uses to install a base operating system on a machine from external repository. Installation media must be in the format of an  +operating system installation tree, and must be accessible to the machine hosting the installer through an HTTP URL. You can view installation media by navigating to *Hosts* > *Installation +Media* menu.
```
Comment (spetrosi) [inline guides/doc-Provisioning_Guide/topics/Resources.adoc:204]:
```suggestion
Installation media are sources of packages that {ProjectServer} uses to install a base operating system on a machine from an external repository. Installation media must be in the format of an operating system installation tree, and must be accessible to the machine hosting the installer through an HTTP URL. You can view installation media by navigating to *Hosts* > *Installation Media* menu.
```
Add the "an" article and remove redundant + symbols.

File: guides/doc-Provisioning_Guide/topics/Resources.adoc [hunk 2]
```diff
@@ -198,6 +198,54 @@ To view the kickstart tree enter the following command:
...
+
+{Project} comes with a number of installation media pre-defined to official Linux distribution mirrors. The *Path* should be modified to point to the closest mirror or local copy in order to achieve better download performance when installing multiple hosts. Some installation media can be only used with specific versions, for example *CentOS mirror (7.x)* must be used for CentOS 7 or older and *CentOS mirror (8.x)* must be used for CentOS 8 or higher.
```
Comment (spetrosi) [inline guides/doc-Provisioning_Guide/topics/Resources.adoc:206]:
```suggestion
By default, {Project} includes installation media for some official Linux distributions. Note that some of those installation media are targeted for a specific version of an operating system. For example *CentOS mirror (7.x)* must be used for CentOS 7 or later, and *CentOS mirror (8.x)* must be used for CentOS 8 or later.

If you want to improve download performance when using installation media to install operating systems on multiple host, you must modify the installation medium's *Path* to point to the closest mirror or a local copy. 
```
A suggestion to add structure and improve style.

Comment (lzap) [reply guides/doc-Provisioning_Guide/topics/Resources.adoc:206]: "For example *CentOS mirror (7.x)* must be used for CentOS 7 or later" should be "For example *CentOS mirror (7.x)* must be used for CentOS 7 or earlier". I am gonna fix that once I merge your change.

**Comment by lzap** [comment]
Rebased, squashed your changes atop.

**Pull request merged**